go-controller/pkg/factory/factory.go

```diff
@@ -550,26 +550,26 @@ func (wf *WatchFactory) GetNamespaces() ([]*kapi.Namespace, error) {
 	return namespaceLister.List(labels.Everything())
 }
 
-func (wf *WatchFactory) NodeInformer() cache.SharedIndexInformer {
-	return wf.informers[nodeType].inf
+func (wf *WatchFactory) NodeInformer() v1coreinformers.NodeInformer {
+	return wf.iFactory.Core().V1().Nodes()
 }
 
 // LocalPodInformer returns a shared Informer that may or may not only
 // return pods running on the local node.
-func (wf *WatchFactory) LocalPodInformer() cache.SharedIndexInformer {
-	return wf.informers[podType].inf
+func (wf *WatchFactory) LocalPodInformer() v1coreinformers.PodInformer {
```

Comment:
shouldn't we use this opportunity to consume directly the client-go informers and avoid this extra layer?

Reply:
maybe? @squeed had mentioned that there was a memory impact to starting multiple `SharedInformerFactory`'s so i decided to err on the side of caution and pull in this commit from #2131

Reply:
correct - we really really don't want multiple Pod informers. The Pod store is one of the most expensive, memory-wise.